packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -275,7 +336,7 @@ export class NotificationBox extends React.Component<Props, State> {
       ...this.generalNotifications(),
     ]
 
-    if (!notifications || !notifications.length) {
+    if (!notifications.length) {
```

Comment:
If notifications is null will this throw an error as property .length won't exist?

Reply:
It's hidden in the diff but right above this notifications is defined as `[...this.incomingPaymentRequestsNotification(), all the others]` so it should always be defined as an array